src/ontology/mondo-ingest.Makefile

```diff
@@ -169,6 +169,28 @@ update-jinja-sparql-queries:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+# TODO: Add .PHONY here and below to top of file?
+# TODO: need a purl for this?
+# TODO: use this target: tmp/mondo.sssom.tsv
+.PHONY: mondo-mappings
+MONDO_MAPPINGS_TSV = 'https://raw.githubusercontent.com/monarch-initiative/mondo/master/src/ontology/mappings/mondo.sssom.tsv'
+mondo-mappings:
+	wget ${MONDO_MAPPINGS_TSV} -P ../analysis/excluded_terms_in_mondo/ignored/mappings/
```

Comment:
- Use `$(TMPDIR)` instead of instead of /ignored/
- `${MONDO_MAPPINGS_TSV}` should be `$(MONDO_MAPPINGS_TSV)`, but independent of this, as discussed, use `tmp/mondo.sssom. tsv` which is already there.

Reply:
yep, these changes are on the way

Reply:
Yep! I saw the comment on the first bit, and I remember bringing up the mondo mappings yesterday; it will be done.